### ISSUE TITLE
Illuminate\Database\QueryException

### ISSUE THREAD

**Issue by aebada90** [Attendize/Attendize#960]
I get the following error after I installed the website using GIT on Hostinger
SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known (SQL: select count(*) as aggregate from `accounts` where `accounts`.`deleted_at` is null)

Here is the link:http://events.ehopn.com/login

**Comment by quentincaffeino** [comment]
Open `.env` file on your hosting and check if it has correct database credentials

**Comment by engahmed** [comment]
http://events.ehopn.com/signup
I got this error when I regsieter
![image](https://user-images.githubusercontent.com/25064051/129633663-9f479b1c-df5b-497b-b315-eb657a7bc219.png)

**Comment by justynpride** [comment]
@engahmed I agree with the comment from @quentincaffeino. Please can you check your .env file. I've found this issue occurs when the database info in the .env file is not correct or has been reset somehow.

**Comment by quentincaffeino** [comment]
@justynpride from the last message I'm almost certain that .env should be correct since it is able to connect to the database.

@engahmed I think that your database failed to initialize for some reason. Table `timezones` is missing id `30`. Try running [`php artisan migrate:fresh --seed`](https://laravel.com/docs/8.x/seeding#running-seeders). If you don't have access to server ssh try reinstalling Attendize. The problem could be with installing from git, or the fact that you are installing non-release version. Try manually downloading [latest](https://github.com/Attendize/Attendize/releases/latest) release.

**Comment by engahmed** [comment]
I removed everything and used FTP to use the last version. Now I am getting an error about a missing file
Look at : http://events.ehopn.com/

![image](https://user-images.githubusercontent.com/25064051/129711624-b545152e-4756-4111-810a-01e8cabb3388.png)

**Comment by quentincaffeino** [comment]
You need to install dependencies using composer and because you don't have ssh access you need to do that locally and upload website with dependencies (vendor folder).

**Comment by engahmed** [comment]
This is only the file where I have to change these values, right?
![image](https://user-images.githubusercontent.com/25064051/129728817-6546286d-796a-46a1-aec0-5b025e1116d3.png)

**Comment by quentincaffeino** [comment]
APP_URL should be your domain name, for you it will be `http://events.ehopn.com`

**Comment by engahmed** [comment]
Should I change something in the database?
![image](https://user-images.githubusercontent.com/25064051/129734296-d972a67f-a808-49a8-92e8-c5e0f03e534e.png)

**Comment by samdb** [comment]
No, don't edit this file. The database.php file uses the variables from .env or the fallback value listed after it.

Depending on characters are in your password, sometimes with .env variables it helps to wrap the password in quote marks.

**Comment by quentincaffeino** [comment]
For some reason this code isn't executed for you
https://github.com/Attendize/Attendize/blob/23c2cf06b676ec59f349e078e8245e270e688893/app/Http/Controllers/InstallerController.php#L245-L247

Could you please dump your database and send it here? You can omit users tables if you are afraid of password leak.

**Comment by engahmed** [comment]
DB_CONNECTION=mysql
DB_HOST=localhost
DB_PORT=3306
DB_DATABASE=u567388274_event
DB_USERNAME=u567388274_event
DB_PASSWORD=C$CQ/Agu*v1

**Comment by engahmed** [comment]
Here you are
[u567388274_event.zip](https://github.com/Attendize/Attendize/files/7000797/u567388274_event.zip)

**Comment by engahmed** [comment]
As I understood, I have to change only the DB parameters inside env files, please correct me if I am mistaken.

**Comment by quentincaffeino** [comment]
Correct.

As I've said in the previous message, for some reason it doesn't execute seeds for you and that is definitely a bug. Could you please send us your server logs?

**Comment by engahmed** [comment]
Where can I find it?

**Comment by quentincaffeino** [comment]
> Where can I find it?

https://www.hostinger.com/tutorials/website/how-to-check-your-website-error-log

If it is disabled:
1. Enable it
2. Delete `installed` file
3. Complete installation once again
4. Upload log file

**Comment by engahmed** [comment]
Yes, it was disabled. do you mean to remove everything and then upload the source again?

**Comment by quentincaffeino** [comment]
No, it is enough to remove file with name `installed`

**Comment by engahmed** [comment]
I am installing it again  like this
![image](https://user-images.githubusercontent.com/25064051/129754172-2c34b63b-a251-421c-8e40-64dc0d6d902d.png)

**Comment by engahmed** [comment]
Here is the folder of the source after installation containing everything, please have a look
https://drive.google.com/drive/folders/1E5r6bq_kph196csUkVz_RMi6M0E6RWC9?usp=sharing

**Comment by engahmed** [comment]
Is it possible to skip this timezone check?
![image](https://user-images.githubusercontent.com/25064051/130057323-d8dc080a-8c7c-4a9b-9adf-50014ad4b9ce.png)

**Comment by quentincaffeino** [comment]
There's nothing in the logs.

> Is it possible to skip this timezone check?

No, I don't think so.

As a temporary solution here's dump from my timezones table, even though this dump is from postgres database I stripped everything pg related so it should work no problem. Import this into your database.
[timezones.zip](https://github.com/Attendize/Attendize/files/7013918/timezones.zip)

**Comment by engahmed** [comment]
Unfortunaltely it is still showing the same error
http://events.ehopn.com/signup

**Comment by engahmed** [comment]
Can you upload your sourcecode here? So I can update my sourcecode with yours

**Comment by quentincaffeino** [comment]
My code isn't very much different from yours, my stack is however. I run everything in docker with custom image with swoole and postgres.

Can you confirm with a screenshot or something that after importing timezones, corresponding table is populated with data? Or send your database dump once again.

**Comment by engahmed** [comment]
Here
[u567388274_event.zip](https://github.com/Attendize/Attendize/files/7014090/u567388274_event.zip)

**Comment by quentincaffeino** [comment]
> Unfortunaltely it is still showing the same error
Error is different now

Here are data from currencies table.
[currencies.zip](https://github.com/Attendize/Attendize/files/7014174/currencies.zip)

**Comment by engahmed** [comment]
Thanks you very much, it is working now

**Comment by engahmed** [comment]
But now another error came 
http://events.ehopn.com/organiser/1/events?first_run=1
![image](https://user-images.githubusercontent.com/25064051/130067297-faaf6a60-7acc-4d8f-b54f-96144d45d5cd.png)

**Comment by engahmed** [comment]
The latest version of the DB

[u567388274_event.zip](https://github.com/Attendize/Attendize/files/7014301/u567388274_event.zip)

**Comment by engahmed** [comment]
Any solution?

**Comment by quentincaffeino** [comment]
There is not enough info, it's hard to judge based on this screenshot. Upload full error.

**Comment by engahmed** [comment]
Here is the link on login, I got the error
http://events.ehopn.com/organiser/1/events?first_run=1

**Comment by quentincaffeino** [comment]
@engahmed did you change the code? Line `72` in `resources/views/Shared/Layouts/Master.blade.php` is different from this project code.
![image](https://user-images.githubusercontent.com/2855777/130480202-27fc4cf5-a49d-433f-a30d-4033aa7fa2de.png)

https://github.com/Attendize/Attendize/blob/23c2cf06b676ec59f349e078e8245e270e688893/resources/views/Shared/Layouts/Master.blade.php#L71-L73

**Comment by engahmed** [comment]
Nope, I did not change.

**Comment by engahmed** [comment]
I got this error when I clicked on this button
![image](https://user-images.githubusercontent.com/25064051/130781292-b01b4dc3-e6d6-4bd6-8769-5652ff0d0529.png)

**Comment by engahmed** [comment]
![image](https://user-images.githubusercontent.com/25064051/130781371-323283c2-8406-42d8-917b-22b679bf5c79.png)

**Comment by qlickit** [comment]
I know this issue has been abandoned, but I'd still like to reply. After reading this, I have a feeling that your shared host is not really capable of running Attendize and you should move on to something more (semi-)private.
